linux/tests/threadpool_linux_ut/threadpool_linux_ut.c

```diff
@@ -1279,15 +1268,12 @@ TEST_FUNCTION(on_timer_callback_does_nothing_with_null_pointer)
 
 TEST_FUNCTION(threadpool_create_work_item_with_NULL_threadpool_fails)
 {
-    // arrange
```

Comment:
>arrange [](http://example.com/codeflow?start=7&length=7)

nit: I think by convention we try to keep the "arrange" comment even if there is nothing that needs to be done under it

Reply:
Fixed